### ISSUE TITLE
Where have all the Mollusca gone?

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#634]
There are only 792 name usages from WoRMS Mollusca in CoL March 2024 edition ?!?!

**Comment by yroskov** [comment]
Woops! Many thanks, Ari.

**Comment by yroskov** [comment]
@mdoering, something strange happened. PREVIEW and portal show full statistics for [WoRMS Mollusca](https://www.catalogueoflife.org/data/dataset/1130) with 86,756 + 37,640 extinct spp:

![image](https://github.com/CatalogueOfLife/data/assets/28574252/ea99a95f-5f1d-494e-a85c-2d180ab15db0)
 
But actual species are missing in CoL.

**Comment by yroskov** [comment]
**The reason of the problem is that two attempts of sync of 2024-03-03 & 2024-03-03 were cancelled.**

![image](https://github.com/CatalogueOfLife/data/assets/28574252/c5825af2-ee05-4e50-98a0-727a79bbd65f)

**Comment by yroskov** [comment]
**WoRMS Mollusca re-synced 2024-03-26** (at least, sync was started)

**Comment by yroskov** [comment]
@mdoering, how we can check which other syncs were possibly were cancelled in March?

**Comment by mdoering** [comment]
https://www.checklistbank.org/catalogue/3/sector/sync?limit=25&offset=0&state=canceled&state=failed

**Comment by mdoering** [comment]
wanna do another "march" release then?

**Comment by yroskov** [comment]
Yes, we should do it. 

Do you know which other GSDs we need to re-synced because of cancellation without successful later sync? (Sorry, I failed to find it from here https://www.checklistbank.org/catalogue/3/sector/sync?limit=25&offset=0&state=canceled&state=failed. Too much data, and I cannot sort them for dates/period.

**Comment by yroskov** [comment]
>>(Sorry, I failed to find it from here https://www.checklistbank.org/catalogue/3/sector/sync?limit=25&offset=0&state=canceled&state=failed. Too much data, and I cannot sort them for dates/period.

Explaining further "blindness" of this report https://www.checklistbank.org/catalogue/3/sector/sync?limit=25&offset=0&state=canceled&state=failed:

FLOW failed but data are OK in CoL
Alucitoidea - latest sync was successful, and re-sync is not needed
WoRMS Millibase - latest sync was successful, and re-sync is not needed
WoRMS Isopoda indeed needs re-sync = started 2024-03-26
WoRMS Ceriantharia - latest sync was successful, and re-sync is not needed
WoRMS Myxozoa - latest sync was successful, and re-sync is not needed
etc. 

I need to confirm sync of each GSDs "manually".

It would be helpful to have a report on cancelled & failed LATEST syncs.

**Comment by mdoering** [comment]
Agree, but its the first time we have these problems. Might be able to integrate that into the sources overview. Or maybe simpler and more powerful a new filter option in the syncs tab that only shows the last sync for each sector

**Comment by yroskov** [comment]
It looks like, 3 cancelled GSDs need now re-sync: WoRMS Mollusca, WoRMS Isopoda & WoRMS Bryozoa. Syncs started 2024-03-26

**Comment by yroskov** [comment]
Dear @aoern, we uploaded a new version of March today. Could you please check it for possible failures again?

**Comment by aoern** [comment]
Dear @yroskov, I'll do it. As a matter of fact, I interrupted my check after finding this Mollusca issue.
